README.md

```diff
@@ -181,7 +181,13 @@ For example, you can pull the image with latest version by:
 docker pull public.ecr.aws/aws-observability/aws-for-fluent-bit:latest
 ```
 
-If you see errors for image pull limits, try log into public ECR with your AWS credentials:
+If you see errors for image pull limits, or get the following error:
```

Comment:
Do we care to suggest running `docker logout public.ecr.aws` prior to this command or just re-login (as we know it has worked)?

Reply:
I think login is better if you have an AWS account because IIRC there are pull limits outside of AWS datacenters... and login fixes that..